src/RahulH/CoreJavaAssignment3/JavaDecisionMakingAndSwitchcase/Fourth.java

```diff
@@ -0,0 +1,52 @@
+package RahulH.CoreJavaAssignment3.JavaDecisionMakingAndSwitchcase;
+
+import java.util.Scanner;
+
+public class Fourth {
+
+	int a=5;
+	int b=10;
+	int ans;
+	
+	void Calculator(String symbol)
+	{
+		switch (symbol){
```

Comment:
better to wrote some String case or number
n method name's first letter should be small character.